repos/system_upgrade/el9toel10/actors/krb5conf/convertkrb5conf/libraries/convertpamuserdb.py

```diff
@@ -0,0 +1,21 @@
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.stdlib import api
+from leapp.models import OutdatedKrb5confLocation
+
+
+def _convert_krb5conf(conf_file):
+    with open(conf_file) as f:
+        text = f.read().replace('/etc/ssl/certs/ca-certificates.crt',
+                                '/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem')
+    with open(conf_file, 'w') as f:
+        f.write(text)
+
+
+def process():
+    msg = next(api.consume(OutdatedKrb5confLocation), None)
+    if not msg:
+        raise StopActorExecutionError('Expected OutdatedKrb5confLocation, but got None')
```

Comment:
Prevent stopping the upgrade process in this phase. Just log the error and end the actor.
```suggestion
    if not msg:
        api.current_logger().error(
            'Expected OutdatedKrb5confLocation, but got None. '
            'Cannot apply possibly needed changes in kerberos configuration files.'
        )
        return
```